### DISCUSSION TITLE
24.7.8

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1849] [General]
## Releases Notes for 24.7.8
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.7.8/draw.io-24.7.8-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v24.7.8/draw.io-24.7.8-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v24.7.8/draw.io-universal-24.7.8.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v24.7.8/drawio-amd64-24.7.8.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v24.7.8/drawio-x86_64-24.7.8.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v24.7.8/drawio-x86_64-24.7.8.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Uses electron 32.0.1
- Updates to [draw.io core 24.7.8](https://github.com/jgraph/drawio/blob/v24.7.8/ChangeLog). All changes from 24.7.6 to 24.7.8 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v24.7.8'>24.7.8</a>.</em>

**Comment by jimmyb-gh** [comment]
Might be the greatest thing since graphical desktops came to be!  Well done devs!
